.github/workflows/publish.yml

```
@@ -0,0 +1,43 @@
+name: Publish Python 🐍 distributions 📦 to PyPI and TestPyPI
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  build-n-publish:
+    name: Build and publish Python 🐍 distributions 📦 to PyPI and TestPyPI
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@master
+      with:
+        fetch-depth: 0
+    - uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+
+    - name: Install python-build and twine
+      run: python -m pip install build twine
+
+    - name: Build package
+      run: python -m build --sdist --wheel .
+
+    - name: List result
+      run: ls -l dist
+
+    - name: Check long_description
+      run: python -m twine check dist/*
+
+    - name: Test package
+      run: |
+        cd ..
+        python -m venv testenv
+        testenv/bin/pip install pytest pytest-doctestplus/dist/*.whl
+        testenv/bin/pytest pytest-doctestplus/tests --doctest-plus --doctest-rst
+
+    - name: Publish distribution 📦 to PyPI
+      uses: pypa/gh-action-pypi-publish@master
```

Is it safer to use a "stable" tag?
```suggestion
      uses: pypa/gh-action-pypi-publish@v1
```

---

Yeah probably.

---

Hmm, I run temporarily the workflow on pull request event, and get "unable to find version `v1`".

---

Hmm! Maybe they don't support it. Now that I look at their README, it does say `@master`. Sorry for the noise!